frontend/packages/console-dynamic-plugin-sdk/docs/api.md

```diff
@@ -2077,6 +2078,27 @@ Component that allows to receive contributions from other plugins for the `conso
 
 
 
+---
+
+## `NamespaceBar`
+
+### Summary 
+
+A component that renders a namespace dropdown menu.
+
+
+
+
+### Parameters
+
+| Parameter Name | Description |
+| -------------- | ----------- |
+| `isDisabled` | (optional) A boolean flag that disables the dropdown if set to true |
+| `onNamespaceChange` | (optional) A function that is executed when a namespace option is selected. The only argument it should accept is new namespace as a string. |
+| `children` | (optional) Elements to render below the namespace bar |
```

Comment:
I'm assuming the namespace bar wraps the content, which is why children are needed? We need to make this more clear in the doc.

Reply:
I updated the description to address this.